kie-dmn-feel/src/main/java/org/kie/dmn/feel/runtime/functions/ConcatenateFunction.java

```diff
@@ -39,9 +44,10 @@ public List apply(@ParameterName("list") Object[] lists) {
             } else if ( list != null ) {
                 result.add( list );
             } else {
-                return null;
```

Comment:
@tarilabs , my understanding is that "null" is not a valid parameter for the vast majority of cases. Even it's type is undefined (a bit different than java). That is why the initial implementation returns null if one tries to concatenate anything with null. The same apply for most functions.